Jenkinsfile_CNP

```diff
@@ -29,6 +32,31 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
 }
 
 withPipeline(type, product, component) {
+  after('akschartsinstall'){
+    // Vars needed for AKS testing
+    env.S2S_NAME = 'bulk_scan_payment_processor'
+    env.S2S_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+    env.CORE_CASE_DATA_API_URL = 'http://ccd-data-store-api-aat.service.core-compute-aat.internal'
+    env.IDAM_API_URL = 'https://idam-api.aat.platform.hmcts.net'
+    env.IDAM_CLIENT_REDIRECT_URI = 'https://bulk-scan-orchestrator-aat.service.core-compute-aat.internal/oauth2/callback'
```

Comment:
i think redirect uri should be pointing to `payment processor` not `orchestrator`